source/keywords.rst

```diff
@@ -1,13 +1,131 @@
 Key Words
 ************
-* Variable
-* Assignment
-* Interpreter
-* Namespace
-* Boolean
-* Conditional Flow Control
-* Syntax
-* Parentheses
-* Terminal
-* Execute
-* Terminate
+Variable
+==========
+A variable can be thought of as a labelled box. You store a value in the box
+then you can refer to them later. When you assign a value to a variable you are changing
+what is stored in it.
+
+For example:
+x = 5
+x is the variable.
+
+Assignment
+==========
+In coding assignment is giving a value to a variable.
+
+For example:
+x = 5
+5 is assigned to the variable x
+
+Interpreter
+==========
+An interpreter takes a program written in a language that is human readable
```

Comment:
Is it worth adding a reason why we use languages with Python instead of writing machine code?